src/ol/featureloader.js

```diff
@@ -19,7 +20,14 @@ ol.FeatureLoader;
 
 
 /**
- * @param {string} url Feature URL service.
+ * @api
+ * @typedef {function(ol.Extent, number, ol.proj.Projection) : string}
```

Comment:
It would be good to have a bit more documentation here.  The argument types don't make it through to the API docs (e.g. see http://openlayers.org/en/master/apidoc/ol.html#.TileUrlFunctionType), so some additional prose is helpful.